tests/keras/test_keras_model_export.py

```diff
@@ -44,42 +44,43 @@ def predicted(model, data):
 
 def test_model_save_load(model, data, predicted):
     x, y = data
-    with TempDir(chdr=True, remove_on_exit=True) as tmp:
-        path = tmp.path("model")
-        mlflow.keras.save_model(model, path)
+    tmp = tempfile.mkdtemp()
+    path = os.path.join(tmp, "model")
+    mlflow.keras.save_model(model, path)
 
-        # Loading Keras model
-        model_loaded = mlflow.keras.load_model(path)
-        assert all(model_loaded.predict(x) == predicted)
+    # Loading Keras model
+    model_loaded = mlflow.keras.load_model(path)
+    assert all(model_loaded.predict(x) == predicted)
 
-        # Loading pyfunc model
-        pyfunc_loaded = mlflow.pyfunc.load_pyfunc(path)
-        assert all(pyfunc_loaded.predict(x).values == predicted)
+    # Loading pyfunc model
+    pyfunc_loaded = mlflow.pyfunc.load_pyfunc(path)
+    assert all(pyfunc_loaded.predict(x).values == predicted)
 
 
 def test_model_log(model, data, predicted):
```

Comment:
nit: here we can use the `tracking_uri_mock` fixture from `tests/projects/utils.py`, which sets the tracking URI to a tempdir for us